src/types/vendors.ts

```diff
@@ -53,3 +53,107 @@ export type UpdateVendorPayload = {
   tags?: string[]
   comment?: string
 }
+
+export type SkillsData = {
+  id: string
+  name: string
+}
+
+export type GetSkillsPayload = {
+  data?: SkillsData[]
+}
+
+export interface Prices {
+  vendor_id: string | undefined
+  skill_id: string
+  src_lang_classifier_value_id: string
+  dst_lang_classifier_value_id: string
+  character_fee: number
+  word_fee: number
+  page_fee: number
+  minute_fee: number
+  hour_fee: number
+  minimal_fee: number
+}
+
+export type UpdatePricesPayload = {
+  data?: Prices[]
+}
+
+export type PricesData = {
+  id: string
+  vendor_id: string | undefined
+  skill_id: string
+  src_lang_classifier_value_id: string
+  dst_lang_classifier_value_id: string
+  updated_at: string
+  created_at: string
+  character_fee: number
+  word_fee: number
+  page_fee: number
+  minute_fee: number
+  hour_fee: number
+  minimal_fee: number
+  vendor: {
```

Comment:
**[blocker]** Let's use existing types if they already exists. We have the "Vendor" type defined under types/vendors